### PR TITLE
LPD-35456 Becoming a Liferay Marketplace publisher

### PR DESCRIPTION
**Pull request by joanagrego**

### CODE REVIEW

File: modules/test/playwright/tests/workspaces/liferay-workspace-marketplace/pages/become-publisher/becomePublisherPage.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	readonly becomePublisherTitle: Locator;
+	readonly completePublisherTitle: Locator;
+	readonly continueButton: Locator;
+	readonly description: Locator;
+	readonly firstName: Locator;
+	readonly lastName: Locator;
+	readonly phone: Locator;
+	readonly requestAccountButton: Locator;
+	readonly requestAccountTitle: Locator;
+	readonly thankYou: Locator;
```
Comment (kevenleone) [inline modules/test/playwright/tests/workspaces/liferay-workspace-marketplace/pages/become-publisher/becomePublisherPage.ts:19]: alphabetical order

File: modules/test/playwright/tests/workspaces/liferay-workspace-marketplace/pages/become-publisher/becomePublisherPage.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+		this.continueButton = page.getByRole('button', {name: 'Continue'});
+		this.description = page.getByPlaceholder('Enter the name and a brief');
+		this.firstName = page.getByPlaceholder('Enter first name');
+		this.lastName = page.getByPlaceholder('Enter last name');
+		this.phone = page.getByPlaceholder('___–___–____');
+		this.requestAccountTitle = page.getByText('Request a Marketplace');
+		this.requestAccountButton = page.getByRole('button', {
+			name: 'Request Account',
+		});
+		this.thankYou = page.getByText('Thank you for your');
```
Comment (kevenleone) [inline modules/test/playwright/tests/workspaces/liferay-workspace-marketplace/pages/become-publisher/becomePublisherPage.ts:34]: alphabetical order

**Review by kevenleone — CHANGES_REQUESTED** [review]
Hi Joana, thank you for all your effort during the last 3 years working together, this is my last review. but all I want to ask (besides my last request 😢 ) is for you to continue being this amazing person and talented professional, was a pleasure to share so many good moments working with you at Liferay.

This is my last LGTM 🔥 (don't forget to make my changes 😄 )

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**